.github/workflows/qa-integrate.yml

```diff
@@ -138,7 +202,8 @@ jobs:
   unit-test-php:
     name: 'Unit test: PHP (v${{ matrix.php }})'
     runs-on: ubuntu-latest
-    needs: [lint-css, lint-js, lint-php]
+    needs: pre-run
+    if: needs.pre-run.outputs.changed-file-count > 0
```

Comment:
Same sentiment here.

Reply:
The concern being that a PHPUnit test may load a CSS file or a JS file?

Reply:
Yes.